gobblin-core/src/main/java/gobblin/publisher/BaseDataPublisher.java

```diff
@@ -59,22 +60,21 @@ public void initialize()
     int branches = this.state.getPropAsInt(ConfigurationKeys.FORK_BRANCHES_KEY, 1);
     // Get a FileSystem instance for each branch
     for (int i = 0; i < branches; i++) {
-      URI uri = URI.create(getState()
-          .getProp(ForkOperatorUtils.getPropertyNameForBranch(ConfigurationKeys.WRITER_FILE_SYSTEM_URI, branches, i),
+      URI uri =
```

Comment:
Are you using the LinkedIn's coding style template and formatting this part using that?